.rubocop_todo.yml

```diff
@@ -0,0 +1,991 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2021-03-01 12:00:00 UTC using RuboCop version 1.11.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 28
+# Cop supports --auto-correct.
+# Configuration parameters: TreatCommentsAsGroupSeparators, ConsiderPunctuation, Include.
+# Include: **/*.gemfile, **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
+  Exclude:
+    - 'Gemfile'
+
+# Offense count: 9
+# Configuration parameters: Include.
+# Include: **/*.gemspec
+Gemspec/RequiredRubyVersion:
+  Exclude:
+    - 'plugins/current_orders/foodsoft_current_orders.gemspec'
+    - 'plugins/discourse/foodsoft_discourse.gemspec'
+    - 'plugins/documents/foodsoft_documents.gemspec'
+    - 'plugins/links/foodsoft_links.gemspec'
+    - 'plugins/messages/foodsoft_messages.gemspec'
+    - 'plugins/polls/foodsoft_polls.gemspec'
+    - 'plugins/printer/foodsoft_printer.gemspec'
+    - 'plugins/uservoice/foodsoft_uservoice.gemspec'
+    - 'plugins/wiki/foodsoft_wiki.gemspec'
```

Comment:
Could this become something like `plugins/*/*.gemspec` in `.rubocop.yml`?
As these plugins are bundled with Foodsoft, they'd have the same minimum version as Foodsoft.

Reply:
done